pip-requirements.txt

```diff
@@ -0,0 +1,2 @@
+mock>=2.0.0
+coveralls
```

Comment:
There's several issues.

1. We should attempt to verify that Mantid is installed. Although we don't want this to interfere with our ability to run the tests against the presenters. 
1. mslice may only work with a specific version of Mantid, so I think it would be important to verify that you have the version of mantid that you expect before mslice actually tries to do anything.
1. If we are relying on an external mantid install, we should document that in the README